src/components/controllers/slot-controller.ts

```diff
@@ -53,22 +53,22 @@ export class HasSlotController implements ReactiveController {
 
     if (
       (this.slotNames.includes('[default]') && !slot.name) ||
-      (slot.name && this.slotNames?.includes(slot.name))
+      (slot.name && this.slotNames.includes(slot.name))
     ) {
       this.host.requestUpdate();
     }
   }
 }
 
-//
-// Given a slot, this function iterates over all of its assigned element and text nodes and returns the concatenated
-// HTML as a string. This is useful because we can't use slot.innerHTML as an alternative.
-//
+/**
+ * Given a slot, this function iterates over all of its assigned element and text nodes and returns the concatenated
+ * HTML as a string. This is useful because we can't use slot.innerHTML as an alternative.
+ */
 export function getInnerHTML(slot: HTMLSlotElement): string {
   const nodes = slot.assignedNodes({ flatten: true });
   let html = '';
 
-  [...nodes].map(node => {
+  [...nodes].forEach(node => {
```

Comment:
Out of curiosity why did you switch from Map to ForEach?

Reply:
In this instance, I actually can't tell you why. This was updated directly from Shoelace, and a couple of its utility functions. 

My guess would be it proved to be more performant in this instance.. But that's literally a guess.